### ISSUE TITLE
Add Localization to the Backend

### ISSUE THREAD

**Issue by gmeben** [MESH-Research/CCR#1465]
### Problem/Why
While website copy is handled by i18n.js on the frontend, there is no translation implemented for website and email notification copy that's handled on the backend. All text on the backend is currently in English and is not consolidated in one location but rather appears throughout files in the `backend` directory.

### Goal/What
Utilize localization in Laravel so that all text can be retrieved from various language files. 

### To Do / Acceptance Criteria

- [ ]

### Review Checklist

- [ ]

### Accessibility Checklist

- [ ]

**Comment by wreality** [comment]
I think we should make a distinction here. /most/ natural language in the backend that’s displayed to the user via the client should be tokenized to be translated in the client. What we need to translate here are things like email templates and such that don’t go through the client of their way to the user. Thoughts?  @gmeben

**Comment by gmeben** [comment]
@wreality Yes, that makes sense to me. I'm imagining that localization will address the need for translation of copy that gets presented in emails. We'll eventually need to capture users' locale setting, but I think we should still have a centralized place to manage all that text in the same way we do for the client even if we just support en-US for the time being.

**Comment by sattler9** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/CCR-Development-5ff39ba363391e0019713c87/issues/MESH-Research/CCR/1465?planning-poker) with Zenhub @gmeben @Sruthin86 @vilagshe @wreality
